src/app/(proper_react)/redesign/(authenticated)/admin/feature-flags/ModifyInputField.tsx

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+export const ModifyInputField = (props: {
+  id: string | undefined;
+  name: string | undefined;
+  defaultValue: string | undefined;
+}) => {
+  // @ts-ignore FIXME determine correct event type
+  const handleChange = void (async (event) => {
+    const id = event.target.id;
+    const name = event.target.name;
+    const value = event.target.value;
+
+    const endpoint = `/api/v1/admin/feature-flags/${name as string}`;
```

Comment:
Possibly just a TypeScript-ism that I'm not familiar with yet, but should this be:
```diff
-const endpoint = `/api/v1/admin/feature-flags/${name as string}`;
+const endpoint = `/api/v1/admin/feature-flags/${name}` as string;
```

Considering earlier we were casting the entire `const` `as string`:
https://github.com/mozilla/blurts-server/blob/ad093cf82259520737ea19e3598c83a1a777c933/src/app/(proper_react)/redesign/(authenticated)/admin/feature-flags/AddFeatureFlag.tsx#L21